pkg/resources/create_resource.go

```diff
@@ -0,0 +1,25 @@
+package resources
+
+import (
+	"context"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// CreateFromYAML creates a resource from a YAML manifest
+func CreateFromYAML(s *runtime.Scheme, cl client.Client, asset []byte) error {
```

Comment:
`asset` param_name not relevant in `CreateFromYAML()` context.  Something like `yamlContent` or `content` or something_else would be better.

Reply:
done in b26c03e